spec/api/policies/complete_endpoint_spec.rb

```diff
@@ -154,7 +154,7 @@ def mutate_json(data)
 
         context "with a payload demonstrating validation edge conditions for 'name'" do
 
-          let(:name_with_all_valid_chars) { 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqurstuvwxyz0123456789-_:.' }
+          let(:name_with_all_valid_chars) { 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqurstuvwxyz0123456789' }
```

Comment:
Think I'd rather we skip this one temporarily; We definitely should allow underscore, hyphen, and period (and might as well allow colon at that point).